docs/docs_source/wopi/files/PutRelativeFile.rst

```diff
@@ -54,34 +54,41 @@ PutRelativeFile
     :code 500: Server error
     :code 501: Unsupported
 
-Required response properties
-----------------------------
+Response
+--------
 
-..  default-domain:: js
+The response to a PutRelativeFile call is JSON (as specified in :rfc:`4627`) containing a number of parameters, some of
+which are optional.
+
+All optional values default to the following values based on their type:
+
+=======  ================
+Type     Default value
+=======  ================
+Boolean  ``false``
+String   The empty string
+=======  ================
 
-The following properties must be present in all PutRelativeFile responses:
 
-..  data:: Name
-    :noindex:
+Required response properties
+----------------------------
 
-    The **string** name of the newly created file without a path.
-    **This is a required value in all PutRelativeFile responses.**
+The following properties must be present in all PutRelativeFile responses:
 
-..  data:: Url
-    :noindex:
+Name
+    The **string** name of the newly created file without a path. **This is a required value in all PutRelativeFile
+    responses.**
 
-    The **string** URI to the :ref:`CheckFileInfo` URI of the newly created file on the host.
-    **This is a required value in all PutRelativeFile responses.**
+Url
+    The :ref:`CheckFileInfo` URI, as a **string**, of the newly created file on the host, including the :term:`file
+    ID` for the new file and a valid :term:`access token`. **This is a required value in all PutRelativeFile
+    responses.**
```

Comment:
Nope, they're not the same thing:

>The `WOPISrc` parameter tells Office Online the URL of the host’s WOPI Files endpoint. In other words, it is a URL of the form `http://server/<...>/wopi*/files/(id)`, where `id` is the file id of the file.

This URL that needs to be returned here is the CheckFileInfo URL, which includes an access token. The WOPISrc never includes an access token, hence the difference.